imu: 
    topic: imu/data
    rate: 60.0
    msgType: Imu
    covariance_matrix:    # 9 entries each
        orientation: 
            - 0.001   # 1 var roll  
            - 0.0000   # 2
            - 0.0000   # 3
            - 0.0000   # 4
            - 0.001   # 5 var pitch
            - 0.0000   # 6   
            - 0.0000   # 7 
            - 0.0000   # 8 
            - 0.001      # 9 var yaw
        
        angular_velocity:
            - 0.006   # 1 var x vel roll  
            - 0.0000   # 2
            - 0.0000   # 3
            - 0.0000   # 4
            - 0.006   # 5 var y vel pitch
            - 0.0000   # 6   
            - 0.0000   # 7 
            - 0.0000   # 8 
            - 0.006     # 9 var z vel yaw
        
        linear_acceleration:
            - 0.006   # 1 var x acc   
            - 0.0000   # 2
            - 0.0000   # 3
            - 0.0000   # 4
            - 0.006   # 5 var y acc
            - 0.0000   # 6   
            - 0.0000   # 7 
            - 0.0000   # 8 
            - 0.006   # 9 var z acc
            
wheel_odom:
    topic: wheel_odom
    rate: 60.0
    msgType: Odometry
    covariance_matrix: 
        pose: 
        - 0.006   # 1 var x pos  
        - 0.0000   # 2
        - 0.0000   # 3
        - 0.0000   # 4
        - 0.0000   # 5
        - 0.0000   # 6   
        - 0.0000   # 7 
        - 0.006   # 8 var y pos
        - 0.0000 # 9
        - 0.0000 # 10
        - 0.0000 # 11
        - 0.0000 # 12
        - 0.0000 # 13
        - 0.0000 # 14 
        - 0.006  # 15 var z pos
        - 0.0000 # 16
        - 0.0000 # 17
        - 0.0000 # 18
        - 0.0000 # 19
        - 0.0000 # 20
        - 0.0000 # 21 
        - 0.05 # 22 var roll
        - 0.0000 # 23
        - 0.0000 # 24
        - 0.0000 # 25
        - 0.0000 # 26
        - 0.0000 # 27
        - 0.0000 # 28 
        - 0.05 # 29 var pitch
        - 0.0000 # 30
        - 0.0000 # 31
        - 0.0000 # 32
        - 0.0000 # 33
        - 0.0000 # 34
        - 0.0000 # 35
        - 0.05 # 36 var yaw
        
        twist:
        - 0.001   # 1 var x vel  
        - 0.0000   # 2
        - 0.0000   # 3
        - 0.0000   # 4
        - 0.0000   # 5
        - 0.0000   # 6   
        - 0.0000   # 7 
        - 0.001   # 8 var y vel
        - 0.0000 # 9
        - 0.0000 # 10
        - 0.0000 # 11
        - 0.0000 # 12
        - 0.0000 # 13
        - 0.0000 # 14 
        - 0.001  # 15 var z vel
        - 0.0000 # 16
        - 0.0000 # 17
        - 0.0000 # 18
        - 0.0000 # 19
        - 0.0000 # 20
        - 0.0000 # 21 
        - 0.001 # 22 var roll ang vel
        - 0.0000 # 23
        - 0.0000 # 24
        - 0.0000 # 25
        - 0.0000 # 26
        - 0.0000 # 27
        - 0.0000 # 28 
        - 0.001 # 29 var pitch ang vel
        - 0.0000 # 30
        - 0.0000 # 31
        - 0.0000 # 32
        - 0.0000 # 33
        - 0.0000 # 34
        - 0.0000 # 35
        - 0.001 # 36 var yaw ang vel

gps:
    topic: pose_gps
    rate: 60.0
    msgType: PoseWithCovarianceStamped
    covariance_matrix: 
        pose: 
        - 0.006   # 1 var x pos  
        - 0.0000   # 2
        - 0.0000   # 3
        - 0.0000   # 4
        - 0.0000   # 5
        - 0.0000   # 6   
        - 0.0000   # 7 
        - 0.006   # 8 var y pos
        - 0.0000 # 9
        - 0.0000 # 10
        - 0.0000 # 11
        - 0.0000 # 12
        - 0.0000 # 13
        - 0.0000 # 14 
        - 0.006  # 15 var z pos
        - 0.0000 # 16
        - 0.0000 # 17
        - 0.0000 # 18
        - 0.0000 # 19
        - 0.0000 # 20
        - 0.0000 # 21 
        - 0.001 # 22 var roll
        - 0.0000 # 23
        - 0.0000 # 24
        - 0.0000 # 25
        - 0.0000 # 26
        - 0.0000 # 27
        - 0.0000 # 28 
        - 0.001 # 29 var pitch
        - 0.0000 # 30
        - 0.0000 # 31
        - 0.0000 # 32
        - 0.0000 # 33
        - 0.0000 # 34
        - 0.0000 # 35
        - 0.001 # 36 var yaw
        

    
